_posts/2016-04-26-cloudready-linux-mint.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Dual-Boot Cloudready and Linux Mint
+tags: linux-mint cloudready dual-boot grub
+username: likethemammal
+---
```

Comment:
oh didnt know that. i thought it was for a post referencing another post, nvm then